action.yml

```diff
@@ -29,25 +29,33 @@ runs:
         shell: bash
       - name: "Install dependencies"
         run: |
-          if [ "${{inputs.ABI}}x" = "32x" ];
-          then
-            sudo dpkg --add-architecture i386
-            sudo apt-get update
-            packages=(
-              libgmp-dev:i386
-              libreadline-dev:i386
-              zlib1g-dev:i386
-              gcc-multilib
-              g++-multilib
-            )
-          else
-            packages=(
-              libgmp-dev
-              libreadline-dev
-              zlib1g-dev
-            )
+          echo "Installing dependencies"
+          if [ "${{runner.os}}" = "Linux" ]; then
+            if [ "${{inputs.ABI}}" = "32" ]; then
```

Comment:
In the future, consider using `[[ ]]` instead of `[ ]` -- the former is bash specific and among other things deals correctly with string comparisons where one string is empty.